transports/quic/src/lib.rs

```diff
@@ -25,29 +25,27 @@
 //! Example:
 //!
 //! ```
-//! # #[cfg(not(feature = "async-std"))]
+//! # #[cfg(not(feature = "tokio"))]
 //! # fn main() {}
 //! #
-//! # #[cfg(feature = "async-std")]
-//! # fn main() -> std::io::Result<()> {
+//! # #[cfg(feature = "tokio")]
+//! # #[tokio::main]
+//! # async fn main() -> std::io::Result<()> {
 //! #
 //! use libp2p_core::{transport::ListenerId, Multiaddr, Transport};
 //! use libp2p_quic as quic;
 //!
 //! let keypair = libp2p_identity::Keypair::generate_ed25519();
 //! let quic_config = quic::Config::new(&keypair);
-//!
-//! let mut quic_transport = quic::async_std::Transport::new(quic_config);
-//!
+//! let mut quic_transport = quic::tokio::Transport::new(quic_config);
 //! let addr = "/ip4/127.0.0.1/udp/12345/quic-v1"
 //!     .parse()
 //!     .expect("address should be valid");
 //! quic_transport
 //!     .listen_on(ListenerId::next(), addr)
 //!     .expect("listen error.");
-//! #
-//! # Ok(())
-//! # }
+//! Ok(())
+//! }
```

Comment:
```suggestion
//! #
//! # Ok(())
//! # }
```

(Revert to how it was before)

Reply:
Just to clarify: I mean re-adding the `#`es here again. This will result in the lines being hidden in the generated rustdoc, like it is also done for the boilerplate code further up that creates the `main` function